src/users/components/UserList.tsx

```diff
@@ -24,6 +32,9 @@ const UserList: FC = () => {
 
   const filteredInvites = filter(invites, ['email', 'role'], searchTerm)
 
+  const isDeletable = user =>
+    selfRemovalFromAccount ? users.length > 1 : user.id !== currentUserId
```

Comment:
if `currentUserId` ends being `undefined` due to `useSelector(getMe)` not returning anything with an id, that is still acceptable right?

Reply:
Yeah that should match the existing behavior from before we added the feature flag, so it should be as safe as it was before.